index.bs

```diff
@@ -1957,6 +1977,11 @@ To <dfn>obtain a randomized source response pick rate</dfn> given a [=randomized
 1. Let |numPossibleValues| be the [=set/size=] of |possibleValues|.
 1. Return |numPossibleValues| / (|numPossibleValues| - 1 + e<sup>|epsilon|</sup>).
 
+To <dfn>obtain pick rate for a source with attribution scope</dfn> given a positve 32-bit integer |numSourceStates|, a positve 32-bit integer |numEventStates|, a positve 32-bit integer |numUsedScopes|, a positive and a double |epsilon|:
```

Comment:
Nitpick -- here we're using just `attributionScopeLimit`, meaning we don't know how many scopes will actually be used?

Reply:
I'm confused - this method doesn't have an iteration.

Reply:
the iteration is in `compute the channel capacity of a source`. `numUsedScopes` is one of the value we pick from [0, k)